components/avatar.css

```diff
@@ -0,0 +1,45 @@
+.avatar-box,
+.avatar-circle {
+	margin: 1rem 1rem;
+}
+.avatar-circle {
+	border-radius: 50%;
+}
+.avatar-img {
+	width: 100%;
+}
+.avatar-text {
+	background-color: var(--color-primary);
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	font-weight: 700;
+	font-size: 1.6em;
+}
+.avatar-icon {
```

Comment:
Can declare common properties together for avatar-icon and avatar-text, they both have almost the same repeating properties. This will lead to fewer lines, better readability.

```
.avatar-icon,
.avatar-text {
       display: flex;
 	align-items: center;
 	justify-content: center;
 	font-size: 1.6em;
 	background-color: var(--color-primary);
}

.avatar-text {
        font-weight: 700;
}
```

Reply:
Yes fixed this! 

> avatar component looks great! 👍🏻
> 
> Just a small feedback for overall project - better naming convention could have been used(for example for margin you have used longer class name, for example, naming convention for margin in utilities.css. Also I see that you have used some random class names like code-classes in main.css which can be name as bg-(some relatable name) as it is just setting background color for text

Changed the classname, Thank you!